samples/nextjs/jss-create.js

```diff
@@ -43,6 +43,8 @@ module.exports = function createJssProject(argv, nextSteps) {
     createEmptyStarter();
   }
 
+  strip({ stripCode: argv.empty });
```

Comment:
Shouldn't this strip call occur _after_ the "Cleaning up the sample..." log below (like you had it previously)? Maybe you can pass argv.empty as a parameter, and do the work there. e.g.

```
setEmpty(argv.empty);
...
function setEmpty(empty) {
  console.log(chalk.cyan('Cleaning up the sample...'));
  strip({ stripCode: empty, suffix: 'EMPTY' });
  if (!empty) {
    return;
  }
  const dataDir = getPath('data');
  ...
}
```

Reply:
Agree with you. Done. Looks better when everything in the same function